oaknational/Oak-Web-Application
# [CUR-940] Switch curriculum visualiser to newer MV

## Description

Music year: 2005

- List of changes

This change consolidates the curriculum visualiser and docs to one MV rather than two.

I've run this through the curriculum visual testing tool and can see no visual changes to units.  

## How to test

1. Go to https://deploy-preview-2853--oak-web-application.netlify.thenational.academy/teachers/curriculum
2. Check the units are correct

## Checklist
